### DISCUSSION TITLE
Qint - Quasar internationalization made easy.

### DISCUSSION THREAD

**Discussion by ejez** [quasarframework/quasar#9342] [Show and Tell]
I made some packages to facilitate apps i18n:

- [Wint](https://github.com/pyxo-dev/wint) (Web apps Int): A library of useful i18n functions.
- [Vint](https://github.com/pyxo-dev/vint) (Vue Int): Extends Wint with Vue related utils.
- [Qint](https://github.com/pyxo-dev/qint) (Quasar Int): Extends Vint with Quasar related utils.

The functions provided by the above packages can be used as standalone functions or through a `Wint | Vint | Qint` instance that is created with a config object. (This will greatly facilitate the use of the functions, as there will be no need to provide the argument options that are already present in the config object).

Api docs are available:
- https://wint.pyxo.net/api
- https://vint.pyxo.net/api
- https://qint.pyxo.net/api

For a Quasar app, you need only to install `Qint` (which will make `Wint` and `Vint` also available). However for a much easier and quicker way to get started you can install the Quasar app extension: [Int](https://github.com/pyxo-dev/quasar-app-extension-int).

Demo: https://qint-demo.pyxo.net

Appreciate your feedback!

**Comment by ejez** [comment]
I want to mention also that the source code is easy to read, not that long or complicated, and contains lots of comments. You are invited to scan through the source code to get the best from the packages.

**Comment by etewiah** [comment]
I am blown away that this project doesn't have more stars.  It is pretty awesome.

So awesome I made this post on hacker news ;)

https://news.ycombinator.com/item?id=28332111

**Reply by ejez** [reply]
many thanks @etewiah , I hope you will find it useful.

**Comment by tarikrital** [comment]
Any JS sample instead of TS?

**Reply by pdanpdan** [reply]
Remove the typing???

**Reply by tarikrital** [reply]
there are many files on the project! SSR & SPA! also I knew Quasar didn't move yet to TS 100%..
